network/p2p/sporking_test.go

```diff
@@ -28,136 +25,149 @@ import (
 //    updating the channel suffix to prevent one-to-K messaging (PubSub) across sporks.
 // 1 and 2 both can independently ensure that nodes from the old chain cannot communicate with nodes on the new chain
 // These tests are just to reconfirm the network behaviour and provide a test bed for future tests for sporking, if needed
-type SporkingTestSuite struct {
-	suite.Suite
-	logger zerolog.Logger
-}
-
-func TestSporkingTestSuite(t *testing.T) {
-	golog.SetAllLoggers(golog.LevelError)
-	suite.Run(t, new(SporkingTestSuite))
-}
```

Comment:
we don't really need a test suit for this, we only share a single logger among tests, which is already provided by the new `nodeFixture`, hence easier to maintain, and even run in parallel.